devicetypes/fibargroup/fibaro-door-window-sensor-2.src/fibaro-door-window-sensor-2.groovy

```diff
@@ -132,7 +134,7 @@ def updated() {
 	if ( settings.temperatureHigh as Integer == 0 && settings.temperatureLow as Integer == 0 ) { 
 		sendEvent(name: "temperatureAlarm", value: null, displayed: false) 

```

Comment:
I'm concerned about this event. I wouldn't expect that we can just send a null value under any circumstances.

Reply:
If the goal is to disable temperature alarm reporting it might be best to just set it to "cleared". I don't think `null` is valid (though, it might work).